plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/debug/ui/launchConfigurations/LaunchConfigurationsDialog.java

```diff
@@ -206,6 +206,12 @@ public String errorMessageWhile() {
 		public String errorMessageUntil() {
 			return "Tree does not has focus or TreeItem '"+item.getText()+"' is not selected";
 		}
+
+		@Override
```

Comment:
making a default impl of getResult to return null will avoid such useless overriding.